vessel-runtime/src/test/java/com/textnow/android/vessel/VesselImplTest.kt

```diff
@@ -141,7 +141,7 @@ class VesselImplTest: BaseVesselTest() {
             name = firstSimple.name,
             number = firstSimple.number?.toDouble()
         )
-        vessel.replace(old = firstSimple, new = replacement)
+        vessel.replace(oldType = firstSimple::class, new = replacement)
```

Comment:
Let's keep the test coverage of the deprecated `replace` too while it still exists